check-list.py

```diff
@@ -223,13 +226,15 @@ def main():
 
                 # Verify that the author matches
                 # TODO: enforce this
```

Comment:
nit: remove the TODO here

Reply:
I left the TODO in because the broadlink adapter still has no author in its non-legacy entry.

And that's why there is no cleanup() call with an author missing

Reply:
Ah, fair point. Merge away.